collector.go

```diff
@@ -73,6 +93,137 @@ var (
 		"The downstream SNR margin in dB",
 		nil, nil,
 	)
+
+	bitswapActiveNearEndDesc = prometheus.NewDesc(
+		prometheus.BuildFQName(namespace, "near_end", "bitswap_active"),
+		"Whether bitswap is active on the near end",
+		nil, nil,
+	)
+	bitswapActiveFarEndDesc = prometheus.NewDesc(
+		prometheus.BuildFQName(namespace, "far_end", "bitswap_active"),
+		"Whether bitswap is active on the far end",
+		nil, nil,
+	)
+	reTxActiveNearEndDesc = prometheus.NewDesc(
+		prometheus.BuildFQName(namespace, "near_end", "retx_active"),
+		"Whether retransmission is active on the near end",
+		nil, nil,
+	)
+	reTxActiveFarEndDesc = prometheus.NewDesc(
+		prometheus.BuildFQName(namespace, "far_end", "retx_active"),
+		"Whether retransmission is active on the far end",
+		nil, nil,
+	)
+	attenuationNearEndDesc = prometheus.NewDesc(
+		prometheus.BuildFQName(namespace, "near_end", "attenuation_db"),
+		"The attenuation on the near end in dB",
+		nil, nil,
+	)
+	attenuationFarEndDesc = prometheus.NewDesc(
+		prometheus.BuildFQName(namespace, "far_end", "attenuation_db"),
+		"The attenuation on the far end in dB",
+		nil, nil,
+	)
+	crcCountNearEndDesc = prometheus.NewDesc(
+		prometheus.BuildFQName(namespace, "near_end", "crc_count"),
+		"The number of CRC errors on the near end",
+		nil, nil,
+	)
+	crcCountFarEndDesc = prometheus.NewDesc(
+		prometheus.BuildFQName(namespace, "far_end", "crc_count"),
+		"The number of CRC errors on the far end",
+		nil, nil,
+	)
+	erroredSecondsNearEndDesc = prometheus.NewDesc(
+		prometheus.BuildFQName(namespace, "near_end", "errored_seconds"),
+		"The number of seconds the near end was errored",
+		nil, nil,
+	)
+	erroredSecondsFarEndDesc = prometheus.NewDesc(
+		prometheus.BuildFQName(namespace, "far_end", "errored_seconds"),
+		"The number of seconds the far end was errored",
+		nil, nil,
+	)
+	severelyErroredSecondsNearEndDesc = prometheus.NewDesc(
+		prometheus.BuildFQName(namespace, "near_end", "severely_errored_seconds"),
+		"The number of seconds the near end was severely errored",
+		nil, nil,
+	)
+	severelyErroredSecondsFarEndDesc = prometheus.NewDesc(
+		prometheus.BuildFQName(namespace, "far_end", "severely_errored_seconds"),
+		"The number of seconds the far end was severely errored",
+		nil, nil,
+	)
+	unavailableSecondsNearEndDesc = prometheus.NewDesc(
+		prometheus.BuildFQName(namespace, "near_end", "unavailable_seconds"),
+		"The number of seconds the near end was unavailable",
+		nil, nil,
+	)
+	unavailableSecondsFarEndDesc = prometheus.NewDesc(
+		prometheus.BuildFQName(namespace, "far_end", "unavailable_seconds"),
+		"The number of seconds the far end was unavailable",
+		nil, nil,
+	)
+	hecErrorCountNearEndDesc = prometheus.NewDesc(
+		prometheus.BuildFQName(namespace, "near_end", "hec_error_count"),
```

Comment:
These look like counters. Should we make these `prometheus.CounterValue`?

The `_count` suffix is reserved for histograms.

```suggestion
		prometheus.BuildFQName(namespace, "near_end", "hec_errors_total"),
```

Reply:
If we think they're counters, they should also end in `_total`.

Reply:
I think it makes sense to model them as monotonically increasing, where the increase() of the metrics would usually be of interest for these (e.g. errored seconds and severely errored seconds).